src/lib/util.ts

```diff
@@ -105,38 +105,53 @@ const midiMetaRegexp: Record<MIDIMetaMappingType, RegExp> = {
 	[MIDIMetaMappingType.ProgramChange]: /^PRGCHG\/(?<prgchg>[0-9]{1,2})$/
 };
 
-export const parseMIDIMappingDisplayValue = (value: string): false | { type: MIDIMetaMappingType, mapping: MIDIMetaMapping } => {
+const parseMIDIByte = (val: string, min: number, max: number): number | null => {
+	if (val === undefined) return null;
```

Comment:
tabbing weird here?

Reply:
don't think so. was just temp b4 I fixed the indent, linter doesn't seem to mind either!?!